src/simulation/LoadGenerator.h

```diff
@@ -42,13 +42,18 @@ enum class LoadGenMode
     // Setup contract instance for Soroban Network Config Upgrade
     SOROBAN_UPGRADE_SETUP,
     // Create upgrade entry
-    SOROBAN_CREATE_UPGRADE
+    SOROBAN_CREATE_UPGRADE,
+    // Blend classic and soroban transactions. Mix of pay, upload, and invoke.
+    BLEND_CLASSIC_SOROBAN,
+    // Setup for blended classic+soroban transactions
+    BLEND_CLASSIC_SOROBAN_SETUP
```

Comment:
Do we need a new mode here? Can we use the existing `soroban_invoke_setup` node?

Reply:
No, `soroban_invoke_setup` is sufficient. I removed `blend_classic_soroban_setup` in 9464b4c52fd72d24347b9cba6b224f04cbdb8eae